photonlib-java-examples/swervedriveposeestsim/src/main/java/frc/robot/Vision.java

```diff
@@ -95,20 +93,21 @@ public PhotonPipelineResult getLatestResult() {
      *     used for estimation.
      */
     public Optional<EstimatedRobotPose> getEstimatedGlobalPose() {
-        var visionEst = photonEstimator.update();
-        double latestTimestamp = camera.getLatestResult().getTimestampSeconds();
-        boolean newResult = Math.abs(latestTimestamp - lastEstTimestamp) > 1e-5;
-        if (Robot.isSimulation()) {
-            visionEst.ifPresentOrElse(
-                    est ->
-                            getSimDebugField()
-                                    .getObject("VisionEstimation")
-                                    .setPose(est.estimatedPose.toPose2d()),
-                    () -> {
-                        if (newResult) getSimDebugField().getObject("VisionEstimation").setPoses();
-                    });
+        Optional<EstimatedRobotPose> visionEst = Optional.empty();
+        for (var change : camera.getAllUnreadResults()) {
+            visionEst = photonEstimator.update(change);
```

Comment:
MULTI_TAG_PNP_ON_COPROCESSOR requires calibration to work, example is misleading cause update is called without calib matricies.

Reply:
Actually, not misleading. It does not require calibration.